app/helpers/alchemy/base_helper.rb

```diff
@@ -1,7 +1,8 @@
 module Alchemy
   module BaseHelper
     def _t(key, *args)
-      I18n.t(key, *args)
+      ActiveSupport::Deprecation.warn("Alchemys `_t` method is deprecated! Use `Alchemy.t` instead.", caller.unshift)
+      Alchemy.t(key, *args)
     end
```

Comment:
This method is now defined both in the `BaseController` and in the `BaseHelper`. Can we get rid of one of them?